aiida/Dockerfile

```diff
@@ -33,14 +33,14 @@ RUN /opt/conda/bin/pip install -r /tmp/requirements.txt && \
 # RENKU_VERSION determines the version of the renku CLI
```

Comment:
I'd exchange everything below this line with the block below https://github.com/SwissDataScienceCenter/renku-project-template/blob/4068f96df5512c082e75766289dad6972a577f3e/python-minimal/Dockerfile#L27.

For sure we need to activate the right virtualenv before un(installing) the renku cli here.

Reply:
I do not think we need this, since the templates are not planned to be used with a dev CLI version
># Install renku from pypi or from github if it's a dev version

And the argument with the pinned renku version can stay that way too.

Reply:
Ok, but even if we always keep it to `RENKU_VERSION=0.16.0` and never install a dev version from github, we _need_ to activate the virtualenv in which the renku cli is installed before pip uninstalling and pip re-installing it. Otherwise we might end up with two renkus, one in `/opt/conda/bin/renku` (installed in this Dockerfile) and one in `/home/jovyan/.renku/bin/renku` which comes with the image.

Reply:
yep, fair enough. I'll add the venv part shortly.